notebook/simu_betterave.ipynb

```diff
@@ -0,0 +1,229 @@
+{
```

Comment:
I suggest we delete this notebook as well. It was a test by importing Allard's tuto notebook. But it brings no more information compared to `02-refactor-leaf-area-index`

Reply:
okay